scanpy/_utils.py

```diff
@@ -570,3 +570,51 @@ def lazy_import(full_name):
         # Make module with proper locking and get it inserted into sys.modules.
         loader.exec_module(module)
         return module
+
+
+# --------------------------------------------------------------------------------
+# Neighbors
+# --------------------------------------------------------------------------------
+
+class NeighborsView:
```

Comment:
Could you explain what this class is meant to be used for?

Reply:
This is a helper class to avoid retrieving different keys for connectivities, distances, dictionary keys and so on. It is not meant to prevent problems with deprecated things in uns['neighbors'].

Reply:
So, generally it's a convenience class for getting accessing graph representations? Could you add this description to a doc-string for the class?

I think it's okay to check if the keys exist, but it shouldn't fall back to checking in `uns`.